genesis/engine/entities/tool_entity/tool_entity.py

```diff
@@ -165,60 +158,66 @@ def substep_pre_coupling_grad(self, f):
 
     def substep_post_coupling(self, f):
         self.update_latest_pos(f)
-        self.update_mesh_pose(f)
+        # self.update_mesh_pose(f)
```

Comment:
Do not keep comments in the code. And if you do, at least add a comment to explain why it is commented.